XSD/language.xsd

```diff
@@ -7,15 +7,33 @@
 	<!-- data element -->
 	<xs:element name="language">
 		<xs:complexType>
-			 <xs:sequence>
+			<!-- deprecated since 5.5 -->
+			<xs:sequence>
 				<xs:element name="category" type="category" maxOccurs="unbounded" />
 			</xs:sequence>
+			<!-- /deprecated since 5.5 -->
```

Comment:
Should this simply be removed, then?

Reply:
No, otherwise IDEs will treat the current structure as erroneous. We can remove it once the current structure is not supported anymore.

Reply:
Treating it as erroneous is not wrong. It's easy to adapt this to the new format and I think it's useful to visibly mark the old structure as deprecated before it's actually removed.